Hybrid-state.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+Oppia currently uses **Protractor** for end-to-end testing but as the Angular team plans to end the development of Protractor at the end of 2022, we are plannning to migrate all these tests to **WebdriverIO**.
+Till the time the migration will be complete the tests will be present in **Hybrid State**, i.e the tests will be present in both webdriverIO and protractor.
+
+There are a few important points about the hybrid state:
+
+* We will have two tools to run the E2E tests i.e protractor and webdriverIO
+
+* Each test suite will be either present in webdriverIO or protractor.
+
+* Same utils files can be present in both webdriverIO and protractor.
+
+* No file will contain the code of both webdriverIO and protractor.
+
+* Protractor suite can only work on protractor utils and vice versa for webdriverIO.
+
+## E2E Migration tracker
+
+The [webdriverio migration tracker](https://docs.google.com/spreadsheets/d/1Mj-llYXMURtis54vpL2VL7BwgRiFIZ1nIFtK3fY3Se4/edit?usp=sharing) holds the record of suites migrated to webdriverIO, suites still in protractor and when I am going to migrate them, dependencies migrated to webdriverIO, dependencies still in protractor, and common dependencies which are present in both protractor and webdriverIO.
+Contributors can use this to get the idea about when a test suite is going to be migrated and hence plan their PR accordingly in order to avoid clashes such as if Shivam Jha opened a PR to migrate a particular suite and the contributor also opens a PR making changes in the same suite at the same time.
+
+## Add/Modify E2E tests
+
+1. **If contributor want to add a new suite:**
+
+* Add the new e2e test suite in webdriverio.
+**Note**: As the first new suite will be added in late August, we will have most of the files already migrated to WebdriverIO like action.js, forms.js, user.js, general.js waitFor.js which are mostly needed for writing a new test.
```

Comment:
All the content associated with the `1` should be indented accordingly. Ditto below

Reply:
You didn't indent far enough. Please always check the rendered markdown to make sure the formatting is right

Reply:
Not needed now, as I removed the text from bullets, as per suggestion received from Shahbaz.